### DISCUSSION TITLE
Has anyone looked into connecting via ComfoClime's WiFi?

### DISCUSSION THREAD

**Discussion by schuon** [michaelarnauts/aiocomfoconnect#38] [Ideas]
Hi all,
so I own a ComfoAir Q + ComfoClime (but no ComfoConnect LAN C). The ComfoClime has a WiFi module which allows the ComfoClime app to control both the ComfoClime and ComfoAir Q. The WiFi module has port 80 open, but I have not looked deeper in sniffing the protocol.
Has anyone looked yet into this?
Cheers

**Comment by schuon** [comment]
Actually it seems quite do-able. It's a standard REST Interface. A call to http://{{system-uuid}}/system/{{system-uuid}}/dashboard for example returns 
`{
  "indoorTemperature": 25,
  "outdoorTemperature": 22.7,
  "exhaustAirFlow": 424,
  "supplyAirFlow": 431,
  "fanSpeed": 2,
  "seasonProfile": 0,
  "temperatureProfile": 0,
  "season": 2,
  "schedule": 1,
  "status": 1,
  "heatPumpStatus": 0
}`

So @michaelarnauts, if you give me some pointers at which point best to abstract / differentiate between the different connection options, I'd go ahead and implement. Then we could have a library that supports both ways and potentially all the upstream tools would work for the new ComfoClime connection route.

**Comment by vslavik** [comment]
Note that there's now a Home Assistant integration using the JSON API: https://github.com/msfuture/comfoclime
